frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -689,8 +689,8 @@ export const surveyLogic = kea<surveyLogicType>([
                     })
                     return survey
                 } catch (error) {
-                    posthog.captureException('Error duplicating survey', {
-                        error,
+                    posthog.captureException(error, {
+                        action: 'duplicateSurvey',
```

Comment:
```suggestion
                        action: 'duplicate-survey',
```

just to follow the pattern